core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -1630,4 +1630,53 @@ object TestUtils extends Logging {
       resource.close()
     }
   }
+
```

Comment:
All the methods below were added so that the new test would compile. They were added to trunk via https://github.com/apache/kafka/commit/6c0aed2fbe282e88fec618e31dcc8417f5f390b1#diff-db06f61adab5c7665b1cc0369eaac846R1630, but it made more sense to copy the relevant TestUtils method than cherry-picking the commit.